stores/current.ts

```diff
@@ -35,6 +43,15 @@ function parseHash(): LchValue | undefined {
   return undefined
 }
 
+export async function loadModel(): Promise<void> {
+  let loader: HTMLDivElement = document.querySelector('.model_loader')!
+  loader.style.display = 'block'
+  await import('../view/model/index.js').then(bundle => {
```

Comment:
Why do you need `.then()` if you have `await`?

```js
let bundle = await import('../view/model/index.js')
```

Reply:
It is unnecessary, removed it